src/Platforms/OraclePlatform.php

```diff
@@ -380,11 +380,11 @@ public function getListSequencesSQL($database)
     /**
      * {@inheritDoc}
      */
-    protected function _getCreateTableSQL($table, array $columns, array $options = [])
+    protected function _getCreateTableSQL($tableName, array $columns, array $options = [])
```

Comment:
Would it make sense to use `$name` for the table name argument? `$columns` and `$options` also belong to the table but don't have this prefix.

Reply:
I've improved the code into something better. Marking as resolved.

Reply:
What do you mean?

Reply:
Sorry, comment was misplaced. I was referring to constraintName padding a space at the end. This one is resolved actually,.